src/utils/link/link-by-prompt.js

```diff
@@ -39,23 +35,16 @@ module.exports = async function linkPrompts(context, flags = {}) {
       type: 'list',
       name: 'linkType',
       message: 'How do you want to link this folder to a site?',
-      choices: LinkChoices,
+      choices: linkChoices,
     },
   ])
 
   let kind
   switch (linkType) {
     case GIT_REMOTE_PROMPT: {
       kind = 'gitRemote'
-      if (repoInfo.error) {
```

Comment:
`GIT_REMOTE_PROMPT` is not added to the choices if `repoInfo.error` is defined so the condition is redundant